app/src/main/res/values/strings.xml

```diff
@@ -307,20 +307,20 @@
     <string name="tab_about">About</string>
     <string name="tab_contributors">Contributors</string>
     <string name="tab_licenses">Licenses</string>
-    <string name="app_description">A free lightweight YouTube frontend for Android.</string>
+    <string name="app_description">A libre lightweight YouTube frontend for Android.</string>
     <string name="contribution_title">Contribute</string>
     <string name="contribution_encouragement">Whether you have ideas of; translation, design changes, code cleaning, or real heavy code changes—help is always welcome. The more is done the better it gets!</string>
     <string name="github_url" translatable="false">https://github.com/TeamNewPipe/NewPipe</string>
     <string name="view_on_github">View on GitHub</string>
     <string name="donation_title">Donate</string>
-    <string name="donation_encouragement">NewPipe gets developed by volunteers which spend their free time to bring the best experience to you. Now it is time to give back to make sure our developers can make NewPipe even more better while enjoying a cup of java!</string>
+    <string name="donation_encouragement">NewPipe is developed by volunteers spending time bringing you the best experience. Give back to help developers make NewPipe even better while enjoying a cup of coffee.</string>
     <string name="donation_url" translatable="false">https://newpipe.schabi.org/donate</string>
     <string name="give_back">Give back</string>
     <string name="website_title">Website</string>
-    <string name="website_encouragement">To get more information and the latest news about NewPipe visit our website.</string>
+    <string name="website_encouragement">Visit the NewPipe Website for more info and news.</string>
     <string name="website_url" translatable="false">https://newpipe.schabi.org/</string>
     <string name="app_license_title">NewPipe\'s License</string>
-    <string name="app_license" translatable="false">NewPipe is Free Software: You can use, study share and improve it at your will. Specifically you can redistribute and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.</string>
+    <string name="app_license">NewPipe is copyleft libre software: You can use, study share and improve it at your will. Specifically you can redistribute and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.</string>
```

Comment:
NewPipe is copyleft libre software

Why copyleft? We have a copyright. We just license it unter GPLv3?

Reply:
I would rather ride free as in freedom.

Reply:
"Free" makes for errors in translation, when people interpret it to mean "gratis software". It will be "free" in every other language, as only English that i know of has this ambiguity.
GPLv3+ is strong copyleft. https://en.wikipedia.org/wiki/Copyleft#Strong_and_weak_copyleft